machinelearning/tensorflow.md

```diff
@@ -265,16 +272,93 @@ export TF_NEED_OPENCL_SYCL=0
 export TF_NEED_COMPUTECPP=0
 export TF_NEED_KAFKA=0
 export TF_NEED_TENSORRT=0
+
+# Configure the build setup. These are gcc build instructions,
+# so, enter "n" for Clang build option.
+# Do you want to use Clang to build TensorFlow? [Y/n]: n
 ./configure
 
 # Issue bazel build command with 'mkl_aarch64' config to enable onednn+acl backend
 bazel build --verbose_failures -s --config=mkl_aarch64  //tensorflow/tools/pip_package:build_pip_package //tensorflow:libtensorflow_cc.so //tensorflow:install_headers
 
 # Create and install the wheel
-./bazel-bin/tensorflow/tools/pip_package/build_pip_package ./wheel-TF2.12.0-py3.8-aarch64
+./bazel-bin/tensorflow/tools/pip_package/build_pip_package ./wheel-TF<version>-aarch64
+
+# The output wheel is generated in /home/ubuntu/tensorflow/wheel-TF<version>-aarch64
+pip install <wheel-TF<version>-aarch64/*.whl>
+```
+
+**Build with Clang**
 
-# The output wheel is generated in /home/ubuntu/tensorflow/wheel-TF2.12.0-py3.8-aarch64
-pip install <wheel-TF2.12.0-py3.8-aarch64/*.whl>
+```
+# Install the llvm version recommended in the TensorFlow release
+# llvm-17.0.2 is recommended for TF v2.16
+cd $HOME
+mkdir llvm_install
+cd llvm_install
+wget https://github.com/llvm/llvm-project/releases/download/llvmorg-17.0.2/clang+llvm-17.0.2-aarch64-linux-gnu.tar.xz .
+tar -xvf clang+llvm-17.0.2-aarch64-linux-gnu.tar.xz clang+llvm-17.0.2-aarch64-linux-gnu/
+sudo cp -r clang+llvm-17.0.2-aarch64-linux-gnu/* /usr
+clang --version
```

Comment:
The below steps are repeat of what we have under `gcc`. Possibly you can combine?